content/about/contact.rst

```diff
@@ -46,7 +33,9 @@ Mailing Addresses
 
 .. class:: no-breaks
 
-  B211 Kerr Administration Building
+  100 Cascade Hall
+
+601 SW 17th Street
```

Comment:
@kelnera This is backwards from what's on http://osuosl.org/contact currently. Does order matter